.github/workflows/first-responder-v2-prs-collect.yml

```
@@ -52,8 +65,31 @@ jobs:
         run: |
           gh pr edit $PR_URL --add-label docs-content-fr
 
+      # Check if the PR is connected to an issue that has the DIY docs label. If yes, then add the DIY docs label to the PR.
+      - name: Check if PR is connected to DIY docs issue
+        id: check-diy-docs
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
```

actionlint (1.7.4)

"github.event.pull_request.body" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions for more details

(expression)